utils/python/CIME/case.py

```diff
@@ -1113,8 +1113,13 @@ def _check_testlists(self, compset_alias, grid_name, files):
 
         tests = Testlist(tests_spec_file, files)
         testlist = tests.get_tests(compset=compset_alias, grid=grid_name)
-        if len(testlist) > 0:
-            logger.info("\nThis compset and grid combination is not scientifically supported, however it is used in %d tests.\n"%(len(testlist)))
+        testcnt = 0
+        for test in testlist:
+            if test["category"] == "prealpha" or test["category"] == "prebeta" or "aux_" in test["category"]:
+                testcnt += 1
+        if testcnt > 0:
+            logger.info("\nThis compset and grid combination is not scientifically supported, however it is used in %d tests.\n"%(testcnt))
```

Comment:
Can we please change the expect False here to simply state:
"WARNING: This compset and grid combination is not test in CESM and therefore is unsupported."
and then exit? Rather than using an expect that gives an error. 
We can move this to a separate PR if this causes too much difficulty here - otherwise I am fine with this PR.